Scripts/build_virtusize_core_xcframework.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
This is the script for generating the latest `VirtusizeCore.xcframework` to add to
https://github.com/virtusize/virtusize-auth-ios/tree/develop/Frameworks